src/main/scala/com/raphtory/core/components/partition/StreamWriter.scala

```diff
@@ -264,6 +277,7 @@ class StreamWriter(
 
   def printUpdateCount() = {
     processedMessages += 1
+    PartitionTelemetry.streamWriterGraphUpdates.inc()
```

Comment:
Again I don't know if we need a total

Reply:
Would possibly be good to add all the remote updates as well though, get an idea of which machines have star nodes and are getting hammered

Reply:
Using remote update adds 👍